tests/check_check_master.c

```diff
@@ -583,40 +636,79 @@ START_TEST(test_check_test_names)
 
   for (i = 0; i < sub_ntests; i++)
   {
-	  char* test_name = get_next_test_name(test_names_file);
+      char* test_name = get_next_test_name(test_names_file);
 
-	  if(test_name == NULL || strcmp(master_tests[i].test_name, test_name) != 0)
-	  {
-		  ck_abort_msg("Expected test name '%s' but found '%s' for test %d:%s",
-				  master_tests[i].test_name,
-				  (test_name == NULL ? "(null)" : test_name),
-				  i, master_tests[i].tcname);
-	  }
+      if(test_name == NULL || strcmp(master_tests[i].test_name, test_name) != 0)
+      {
+          ck_abort_msg("Expected test name '%s' but found '%s' for test %d:%s",
+                  master_tests[i].test_name,
+                  (test_name == NULL ? "(null)" : test_name),
+                  i, master_tests[i].tcname);
+      }
 
-	  free(test_name);
+      free(test_name);
   }
 }
 END_TEST
 
 START_TEST(test_check_all_msgs)
```

Comment:
It did not occur to me that there is a separate test for checking only error messages, then a test for checking all messages (which also include failure messages). Any reason why the failure message only test should simply be removed, as the all messages test should provide equivalent coverage?